GitCommands/Git/AheadBehindDataProvider.cs

```diff
@@ -8,6 +9,7 @@ namespace GitCommands.Git
     public interface IAheadBehindDataProvider
     {
         IDictionary<string, AheadBehindData>? GetData(string branchName = "");
+        void InitLazy();
```

Comment:
```suggestion
        void Initialize();
```

Whether it's lazy or not is an implementation detail.

Reply:
ResetCache() or just Reset may be better then - it is resetting the cached result.